consumer/src/main/kotlin/streams/procedures/StreamsSinkProcedures.kt

```diff
@@ -1,22 +1,21 @@
 package streams.procedures
 
-import org.apache.commons.lang3.StringUtils
 import org.neo4j.logging.Log
 import org.neo4j.procedure.*
 import streams.StreamsEventConsumerFactory
 import streams.StreamsEventSinkConfigMapper
 import streams.StreamsSinkConfiguration
 import java.util.stream.Stream
 
-class StreamResult(@JvmField val event: Any)
+class StreamResult(@JvmField val event: Map<String, *>)
 
 class StreamsSinkProcedures {
 
     @JvmField @Context
     var log: Log? = null
 
     @Procedure(mode = Mode.SCHEMA, name = "streams.consume")
-    @Description("streams.consume(topic, config) - Allows to subscribe custom topics")
+    @Description("streams.consume(topic, config) YIELD event - Allows to consume custom topics")
```

Comment:
perhaps at least add the config keys in the description

Reply:
Done!